elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/controllers/Utils.java

```diff
@@ -10,9 +10,21 @@
 
 import java.util.Map;
 
+import javax.ws.rs.core.HttpHeaders;
+
 public class Utils {
     public static String getApiVersion(Map<String, String> requestHeaders) {
         return requestHeaders.getOrDefault("ApiVersion",
                 requestHeaders.getOrDefault("apiversion", NO_VERSION));  //For tomcat
     }
+
+    public static Map<String, String> removeAuthHeaders(Map<String, String> requestHeaders) {
+        if (requestHeaders.get(HttpHeaders.AUTHORIZATION) != null) {
```

Comment:
We should make a copy of the requestHeaders map (and return it) rather than modify the input map.